### DISCUSSION TITLE
How to modify custom prompt words for Copilot Chat ?

### DISCUSSION THREAD

**Discussion by Qingswe** [community/community#68024] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

How to modify custom prompt words for Copilot Chat,I would like to ask questions in a language other than English, so I would like to add 'Please answer in Chinese' to the prompt words

**Comment by Akash1134** [comment]
Hi @Qingswe ,

Could you please elaborate on what exactly do you mean by "Modify the custom prompts"?

Thanks!
